lib/blockchain/cardano-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ -z "$USER" ]; then
+    USER=`whoami`
+fi
+
+tmpdir=$(mktemp -d -t nix-XXXXXXXXXX)
```

Comment:
Question: tmp folder is used to store files needed during installation but no longer after it passes?

Reply:
yes, because I don't wanna download nix installation files to any "our" folder